core/src/main/java/hudson/scm/EditType.java

```diff
@@ -56,7 +56,7 @@ public String toExportedObject() {
 
     public static final EditType ADD = new EditType("add", "The file was added");
     public static final EditType EDIT = new EditType("edit", "The file was modified");
-    public static final EditType DELETE = new EditType("delete", "The file was removed");
+    public static final EditType DELETE = new EditType("trash", "The file was removed");
```

Comment:
This looks like a breaking change for the remote access API.

Reply:
I've changed to delete again. Any other issue?